File: loops.py
# Loops
"""

for in 


while condition:
    pass



"""

for i in range(1, 11): # range(start = 0, stop, step = 1)
    print(i)
    
nombres = ["Hugo", "Paco", "Luis"]

for i in range(len(nombres)):
    print(nombres[i])
    
for nombre in nombres:
    print(nombre)
    
indice = 0
while indice < len(nombres):
    print(nombres[indice])
    indice += 1 # indice = indice + 1

File: loops.js
// Loops
/* 

for(init; condition; increment){

}

for (indice in collection){

}

for(value of collection){

}

while(condition){

}

do {

} while (condition)

*/

for(let i = 1; i <= 10; i++){
    console.log(i)
}


let nombres = ["Hugo", "Paco", "Luis"]

for(let i = 0; i < nombres.length; i++){
    console.log(nombres[i])
}

for(let indice in nombres){
    console.log(nombres[indice])
}

for(let nombre of nombres){
    console.log(nombre)
}

let i = 0;
while (i < nombres.length){
    console.log(nombres[i])
    i++
}